go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/hashicorp/go-version v1.2.0
 	github.com/imdario/mergo v0.3.8
 	github.com/jlewi/cloud-endpoints-controller v0.0.0-20200604211613-aff0aaad5602
-	github.com/kubernetes-sigs/application v0.8.0
+	github.com/kubernetes-sigs/application v0.8.1
```

Comment:
@PatrickXYS any reason to upgrade application version?  I didn't see following code is related to this?

Reply:
Removed